pytri/__init__.py

```diff
@@ -265,7 +265,7 @@ def scatter(self, data, r=0.15, c=0x00babe, name=None) -> str:
             "colors": c
         }, name=name)
 
-    def graph(self, data, r=0.15, c=0xbabe00, name=None) -> str:
+    def graph(self, data, r=0.15, nc=0xbabe00, lc=0x00babe, name=None) -> str:
```

Comment:
I appreciate I set a bad precedent for this, but maybe replace these with more verbose arg names? `node_color` and `edge_color` would match [the `networkx#draw` API](https://networkx.github.io/documentation/stable/reference/generated/networkx.drawing.nx_pylab.draw_networkx.html#networkx.drawing.nx_pylab.draw_networkx)